syntax = "proto3";

package sample;

message Born2 {
  string date = 1;
  string city = 2;
  string hospital_name = 3;
  string father_name = 4;
}

message Car2 {
  string model_name = 1;
  int32 factory_year = 2;
  string license_plate = 3;
}

message Home2 {
  string address = 1;
  string ownership = 2;
}

message PersonSecond {
  string name = 1;
  Born2 born = 2;
  Car2 car = 3;
  Home2 home = 4;
}

message PeopleSecond {
  repeated PersonSecond people = 1;
}
